examples/protogen1.html

```diff
@@ -0,0 +1,480 @@
+<html>
+	<head>
+		<title>Stijns very first dreem.</title>

```

Comment:
Can you make this more descriptive? Also, can you rename the file to something more explicit, perhaps with the word 'circuit' in it? I think people will understand it better that way...